lib/cocoapods/installer/analyzer.rb

```diff
@@ -217,7 +217,7 @@ def generate_podfile_state
       def update_repositories
         sources.each do |source|
           if source.git?
-            config.sources_manager.update(source.name)
+            config.sources_manager.update(source.name, true)
```

Comment:
Yeah I was thinking about this also, right now this will change the output for everyone in the next minor release. Perhaps we should tie this value with `verbose` being ON.